examples/legacy_basic_reconnect.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import time
+from niveristand.errors import RunError
+from niveristand.legacy import NIVeriStand
+
+# This is an example script to demonstrate the usage of NIVeriStand Python API.
+# This script requires Python 32b(tested on Python 3.9.6) and NIVeriStand Python API.
+# It can be downloaded here: https://niveristand-python.readthedocs.io/en/latest/index.html
+
+# This script deploys example project on the selected GATEWAY and it prints two channel values every 100ms
+# This script requires VeriStand serwer to be running. It is ran automatically after launching VeriStand.
+# You can also start it manually from the command line.
+# Default veristand-server.exe location:
+# "C:\Program Files (x86)\National Instruments\VeriStand 2020\veristand-server.exe"
+# Press Ctrl + C to stop the program.
+
+PROJECTDIR = r"C:\Users\Public\Documents\National Instruments\NI VeriStand 2020\Projects"
+SDF = PROJECTDIR + r"\Example\Sinewave Delay.nivssdf"
+
+GATEWAY = "localhost"  # GATEWAY address
+TARGET = "Controller"
+
+SINE_WAVE = 'Aliases/SineWave'
+LOOP_RATE = 'Targets/Controller/System Channels/Actual Loop Rate'
+# channel adresses can be entered as full paths or aliases
+
+
+def sleep():
+    time.sleep(0.1)
+
+
+wks = NIVeriStand.Workspace2(GATEWAY)  # create workspace object
+try:
+    # print("Running %s" % SDF)
```

Comment:
Does this example require to have the SDF already deployed? Can we uncomment these lines to deploy SDF?

Reply:
Yes, this example requires to have the SDF already deployed and yes, we can uncomment these lines to deploy SDF, but that would be reconnecting to the controller we are already connected to - confusing and unnecessary. I think these commented lines should be deleted, as this is an example to show possible use case of this function.